pkg/apiserver/certificate/cacert_controller.go

```diff
@@ -191,7 +164,7 @@ func (c *CACertController) syncMutatingWebhooks(caCert []byte) error {
 
 func (c *CACertController) syncConversionWebhooks(caCert []byte) error {
 	klog.Info("Syncing CA certificate with CRDs that have conversion webhooks")
-	for _, name := range crdsWithConversionWebhooks {
+	for _, name := range c.caConfig.CrdsWithConversionWebhooks {
```

Comment:
It should be `CRDsWithConversionWebhooks`. Is there any verification for this branch? Many tests should be able to catch this.

Reply:
yes, that's my mistake, I missed the warning. test for this feature branch is under development.

Reply:
done